src/models/script.js

```diff
@@ -87,7 +87,10 @@ const ScriptConstraints = {
   },
   runtime_name: {
     presence: true,
-    inclusion: ['golang', 'nodejs_library_v1.0', 'ruby', 'nodejs_library_v0.4', 'python_library_v4.2', 'python_library_v5.0', 'php', 'swift']
+    inclusion: [
+      'golang', 'nodejs_library_v1.0', 'ruby', 'nodejs_library_v0.4',
```

Comment:
We came to the conclusion that maybe we should remove this validation from the lib :D